File: 2068-check-whether-two-strings-are-almost-equivalent/2068-check-whether-two-strings-are-almost-equivalent.py
class Solution:
    def checkAlmostEquivalent(self, word1: str, word2: str) -> bool:
        c1 = Counter([x for x in word1])
        c2 = Counter([x for x in word2])
        for x in 'qwertyuiopasdfghjklzxcvbnm':
            if abs(c1[x] - c2[x]) > 3:
                return False
        return True

File: 2068-check-whether-two-strings-are-almost-equivalent/2068-check-whether-two-strings-are-almost-equivalent.cpp
class Solution {
public:
    bool checkAlmostEquivalent(string word1, string word2) {
        int N;
        N = word1.length();
        int c1[26] = {};
        int c2[26] = {};
        string s = "qwertyuiopasdfghjklzxcvbnm";
        
        for (auto &x: word1) {
            c1[x-'a']++;
        }
        for (auto &x: word2) {
            c2[x-'a']++;
        }
        
        for (auto &x: s) {
            if (abs(c1[x-'a'] - c2[x-'a']) > 3) {
                return false;
            }
        }
        
        return true;
    }
};